libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -1606,6 +1634,10 @@ bool PbftManager::pushPbftBlock_(SyncBlock &&sync_block, vec_blk_t &&dag_blocks_
 
   db_->savePeriodData(sync_block, batch);
 
+  // Save PBFT sortition threshold per period for reward votes verification
+  auto sortition_threshold = std::min<size_t>(COMMITTEE_SIZE, getDposTotalVotesCount());
+  db_->addPbftSortitionThresholdToBatch(pbft_period, sortition_threshold, batch);
+
```

Comment:
we do not need this as `getDposTotalVotesCount` == final_chain_->dpos_eligible_total_vote_count(dpos_period_) so this is already stored

Reply:
Good catch! you are right!
sortition_threshold always is std::mid(COMMITTEE_SIZE, getDposTotalVotesCount())

Reply:
But final_chain only saved after executed PBFT blocks. PBFT finalized blocks are async with execution.

Reply:
then you need to save just last one or? no need to save it per period

Reply:
I think you are right. We can get from getDposTotalVotesCount == final_chain_->dpos_eligible_total_vote_count(dpos_period_)

Create another PR for testing https://github.com/Taraxa-project/taraxa-node/pull/1710

Reply:
Fixed